x-pack/legacy/plugins/reporting/server/lib/create_queue.js

```diff
@@ -7,31 +7,30 @@
 import { Esqueue } from './esqueue';
 import { createWorkerFactory } from './create_worker';
 import { oncePerServer } from './once_per_server';
-import { createTaggedLogger } from './create_tagged_logger';
-
-const dateSeparator = '.';
+import { LevelLogger } from './level_logger';
+import { createTaggedLogger } from './create_tagged_logger'; // TODO remove createTaggedLogger once esqueue is removed
```

Comment:
It looks too hard to update ESQueue to use a new logger object, so let's keep `create_tagged_logger` just for ESQueue.